src/rgw/rgw_lc.h

```diff
@@ -35,6 +35,13 @@ typedef enum {
   lc_complete,
 }LC_BUCKET_STATUS;
 
+typedef enum {
+  standard_ia = 0,
+  onezone_ia,
+  glacier,
+  undefined,
+} STORAGE_CLASS;
```

Comment:
i have a feeling that we'll want to support storage classes with arbitrary names, so i don't think they should be fixed at compile-time with an enum

Reply:
em.. I've removed the enum def.